services/headless-lms/server/src/controllers/mock_azure_endpoint.rs

```diff
@@ -0,0 +1,44 @@
+use crate::prelude::*;
+
+// GET /api/v0/mock_azure/test
+// POST /api/v0/mock_azure/test
+async fn mock_azure_endpoint() -> ControllerResult<String> {
+    let a = r#"
```

Comment:
make this take web::Data<ApplicationConfiguration> and then `assert!(app_conf.test_chatbot && app_conf.test_mode)`